zulip/integrations/twitter/twitter-bot

```diff
@@ -106,10 +107,10 @@ if all([opts.search_terms, opts.twitter_name]):
     parser.error("You must only specify either a search term or a username.")
 if opts.search_terms:
     client_type = "ZulipTwitterSearch/"
-    CONFIGFILE_INTERNAL = os.path.expanduser("~/.zulip_twitterrc_fetchsearch")
+    SINCE_ID_NAME = "since_id_search_" + opts.search_terms
 elif opts.twitter_name:
     client_type = "ZulipTwitter/"
-    CONFIGFILE_INTERNAL = os.path.expanduser("~/.zulip_twitteruserrc_fetchuser")
+    SINCE_ID_NAME = "since_id_name_" + opts.twitter_name
```

Comment:
I was trying to think of a better way to serialize the parameters to a particular instances of the bot, but didn't come up with anything great.  One idea would be to have the configuration file support having a block for each search, and use the names defined there.  E.g. 

```
[twitter_search_foo]
search_terms = ...

[twitter_search_bar]
twitter_name = ...
```